src/api/resources/sync/push.py

```diff
@@ -0,0 +1,85 @@
+from django.core.exceptions import ObjectDoesNotExist
+
+from api.models import Revision
+
+
+def get_request_method(record):
+    """Get what the request method based on the
+    record's properties.
+
+    :param record: Edits record.
+    :type record: dict
+    :return: Request method
+    :rtype: str
+    """
+    if record.get("_deleted") is True:
+        return "DELETE"
+    elif record.get("_last_revision_num") is None:
+        return "POST"
+    else:
+        return "PUT"
+
+
+def has_conflicts(record_id, last_revision_num):
```

Comment:
Does this need to be a separate function? It's only used in one place, is short, and could be misleading in other contexts (i.e. the `>` comparison only means a conflict in the context of a push)